packages/flutter/lib/src/material/ink_decoration.dart

```diff
@@ -279,13 +279,15 @@ class _InkState extends State<Ink> {
     if (_ink == null) {
       _ink = InkDecoration(
         decoration: widget.decoration,
+        isVisible: Visibility.of(context),
```

Comment:
Can it be extended to parameters like `boundaries` to solve [these issues](https://github.com/flutter/flutter/issues/85256#issuecomment-868179635) at the same time?